spec/unit/provider/package/apt_spec.rb

```diff
@@ -428,6 +428,91 @@ def ubuntu1404downgrade_stubs
       end
     end
 
+    describe "when given a response file" do
```

Comment:
These specs test the logic in the new module that is shared between the Apt and Dpkg package resources. What do you think about collecting this group of specs into a single set of `shared_examples` in the module's spec file, testing their inclusion with `MyDebianPackage{Provider|Resource`, and then using the shared examples here and in dpkg_spec?